dicomImport/matRad_importDicomRTDose.m

```diff
@@ -0,0 +1,73 @@
+function [ resultGUI ] = matRad_importDicomRTDose(ct, rtDoseFiles)
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% matRad function to import dicom RTDOSE data
+% 
+% call
+%   resultGUI = matRad_importDicomRTDose(ct, rtDoseFiles)
+%
+% input
+%   ct:             ct imported by the matRad_importDicomCt function
+%   rtDoseFiles:   	cell array of RTDOSE Dicom files
+%
+% output
+%   resultGUI:      matRad resultGUI struct with different beams. Note that
+%                   the summation (called plan) of the beams is named 
+%                   without subscripts, e.g. physical_Dose.
+%
+% References
+%   -
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Copyright 2015 the matRad development team. 
+% 
+% This file is part of the matRad project. It is subject to the license 
+% terms in the LICENSE file found in the top-level directory of this 
+% distribution and at https://github.com/e0404/matRad/LICENSES.txt. No part 
+% of the matRad project, including this file, may be copied, modified, 
+% propagated, or distributed except according to the terms contained in the 
+% LICENSE file.
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+
+%% import and interpolate dose files
+% number of dosefiles
+numDoseFiles = size(rtDoseFiles,1);
+
+for currDoseRun = 1 : numDoseFiles
+    currDose = rtDoseFiles(currDoseRun,:);
+    itemName = strcat('Item_',num2str(currDoseRun));
+    dose.(itemName) = matRad_interpDoseCube( ct, currDose);
+end
+
+%% put dose information and dose meta information to resultGUI
+
+for currDoseRun = 1 : numDoseFiles
```

Comment:
I am somewhat confused: In this loop over all dose files that you found (and previously interpolated), you basically put all dose cubes at the same position. And you keep overwriting them. So if you have multiple dose files for individual beams, you will just remember the last one, right? Wouldn't it be nice to put all in resultGUI? Didn't you have it that way some time ago?